tensorboard/webapp/alert/views/alert_display_snackbar_container.ng.html

```diff
@@ -14,7 +14,18 @@
 See the License for the specific language governing permissions and
 limitations under the License.
 -->
-<div class="message">{{ alert.localizedMessage }}</div>
+<div class="message">
```

Comment:
Just wondering: Did you consider a solution where the messages can include markup? It would be up to the source of the message to explain how it should be formatted. The source would be responsible for specying <a> tags but would not just be limited to linkifying explicity urls.

Reply:
I had not deeply considered any richer alert UI, beyond text, links, and a followup action button.  Providing a string-based message is relatively simple, and there are a handful of immediate use cases we forsee: alert upon most errors (e.g. fetch runs failed, updating experiment description failed + retry, copied to clipboard would even work).

If we did need to render a more expressive component w/ tags, I would consider whether that should be a one-off modal, since it already requires creating a specific NgComponent.  If there are good reasons to support that in the same Alert ngrx feature, I could imagine us extending the state to store a NgComponent, and the view would use `componentFactory` to instantiate it.

Right now, I'm not aware of any use cases that warrant alerts with custom HTML, but please let me know if there any.